index.js

```diff
@@ -7,7 +7,7 @@ const calculate = (arr) => {
   let operand = resultArray[1]
   let num2 = resultArray[2]
 
-  switch ((resultArray[1])) {
+  switch ((operand)) {
```

Comment:
EXCEPTIONAL job this week.  You went above and beyond the skills you have learned thus far and found a unique way to solve.  Thank you for being my commit standard for the rest of the class and keep it up!